src/components/SpiceList.js

```diff
@@ -8,23 +8,36 @@ class SpiceList extends React.Component {
     search: ""
   }
 
+  handleChange = (evt) => {
+    if (evt.type === "checkbox") {
+      this.setState((prevState) => ({
+        fourStarOnly : evt.checked
+      }))
+    } else {
+      this.setState((prevState) => ({
+        search : evt.value
+      }))
+    }
+  }
```

Comment:
It might be helpful to break this up into multiple functions instead of having one change handler for all the filtering logic, that way you have more control over what goes into state, like this:

```js
  handleSearchChange = search => this.setState({ search: search })

  handleFourStarChange = fourStarOnly => this.setState({ fourStarOnly: fourStarOnly })
```

But I like the idea here, and your code for checking the `evt.type` looks good. One other small thing, since you're passing up the `evt.target`, not the whole event object, it might be better to name the parameter for your `handleChange` function `target` instead of `evt`.